### PR TITLE
4.0.0 Release

### PR DESCRIPTION
**Pull request by phenaproxima**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-## 4.0.0-beta2
 * Security updated Lightning Core to 4.1.0, which updates Drupal core to 8.7.1.
   (SA-CORE-2019-07)
-* Updated Lightning Layout to 2.0.0.
-
-## 4.0.0-beta1
-* Updated Lightning Core to 4.0-beta2, which:
-  * Updated Drupal core to 8.7.0-beta2.
-* Updated Lightning Layout to 2.0-beta1, which:
+* Updated Lightning Layout to 2.0.0, which:
```
Comment (balsama) [inline CHANGELOG.md:5]: Can we add the 3.3.0 CHANGELOG entry and merge it with the 3.3.0-beta1?

**Comment by balsama** [comment]
Missing fixture

**Comment by phenaproxima** [comment]
The fixture is _definitely_ there now. :) Not sure why it wasn't showing up initially, but I'm seeing it now.

**Comment by phenaproxima** [comment]
The failures in the PR build are random-ish errors that are not caused by Lightning functionality and don't represent real breaks (they're CI problems, not product problems).

**Comment by balsama** [comment]
Looks good! Merge away!

**Pull request merged**